src/main/webapp/app/tasks/task-list/task-list.component.html

```diff
@@ -2,8 +2,15 @@
   <mat-list-item *ngFor="let task of tasks" class="mat-elevation-z1">
     <mat-icon mat-list-icon>assignment</mat-icon>
     <h4 mat-line>{{task.name}}</h4>
-    <button mat-icon-button aria-label="Delete task" color="primary">
+    <button [ngClass]="task.done ? 'done-button--selected':'done-button'" mat-icon-button aria-label="Task done"
```

Comment:
If the button is implemented to reflect a toggle value, one could improve the accessibility even more by providing additional ARIA attributes (e.g. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Switch_role)